src/edges_analysis/analysis/sky_models.py

```diff
@@ -91,6 +91,7 @@ def get_index(
         lon: [None, np.ndarray] = None,
         sky_model: [None, SkyModel] = None,
     ) -> np.ndarray:
+        ## to return an array of indices of len (latidute) like the other Index functions. This avoids the error in the beams.py code. 
```

Comment:
I think the neater way of doing this is `return np.ones_like(lat) * self.index`.